packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -190,6 +191,9 @@ export default class TableResize implements EditorPlugin {
     }
 
     private createInserter(tableRect: Rect) {
+        if (this.insertingState == ResizeState.None) {
+            return;
```

Comment:
What does this fix for?

Reply:
Since we ignored the None state in resizeTable method and caused a resizing issue. I think we'd better consider the None state in this method as well. Since in this method, the the logic is below and is bug prone.

if (state == horizontal) {
// do horizontal logic
}
else {
// do vertical logic, even for None state.
}